Changelog.md

```diff
@@ -1,5 +1,85 @@
 ## libsdformat 10.X
 
+### libsdformat 10.6.0 (2021-09-08)
+
+1. Parse URDF continuous joint effort/velocity limits
+    * [Pull request #684](https://github.com/ignitionrobotics/sdformat/pull/684)
+
+1. Add enable_orientation SDF element to imu
+    * [Pull request #651](https://github.com/ignitionrobotics/sdformat/pull/651)
+
+1. Add a codecheck make target
+    * [Pull request #682](https://github.com/ignitionrobotics/sdformat/pull/682)
+
+1. Refactor sdf::readXml
+    * [Pull request #681](https://github.com/ignitionrobotics/sdformat/pull/681)
+
+1. Upgrade cpplint and fix new errors
+    * [Pull request #680](https://github.com/ignitionrobotics/sdformat/pull/680)
+
+1. BUG: add missing plugin element to include
+    * [Pull request #675](https://github.com/ignitionrobotics/sdformat/pull/675)
+
+1. Update codecov badge url
+    * [Pull request #678](https://github.com/ignitionrobotics/sdformat/pull/678)
```

Comment:
we redacted this from the 9.6.1 changelog (see https://github.com/ignitionrobotics/sdformat/commit/32ce16a81e707a5000a13cbec686371689b336f7 and https://github.com/ignitionrobotics/sdformat/pull/691#discussion_r703759778)